Tests/NIOHTTP2Tests/HTTP2InlineStreamMultiplexerTests.swift

```diff
@@ -35,8 +35,15 @@ private struct MyError: Error { }
 typealias IODataWriteRecorder = WriteRecorder<IOData>
 
 extension IODataWriteRecorder {
-    func drainConnectionSetupWrites(mode: NIOHTTP2Handler.ParserMode = .server) throws {
-        var frameDecoder = HTTP2FrameDecoder(allocator: ByteBufferAllocator(), expectClientMagic: mode == .client)
+    func drainConnectionSetupWrites(
+        mode: NIOHTTP2Handler.ParserMode = .server,
+        maximumSequentialContinuationFrames: Int
```

Comment:
Can we just default this to 5? Doing that means we can avoid all the other changes in this file.